### ISSUE TITLE
Nginx Proxy Manager fails probes due to slow start

### ISSUE THREAD

**Issue by krojew** [truenas/charts#2482]
NPM seems to be configured differently than the official image and takes a lot of time to start, which results in constant failures of liveness probes. By a lot, I mean tens of minutes. The issue might be related to https://github.com/NginxProxyManager/nginx-proxy-manager/issues/3154, which on one hand is an upstream bug, but on the other is not, because using the official image installed using the quick start instructions at https://github.com/NginxProxyManager/nginx-proxy-manager?tab=readme-ov-file#quick-setup starts immediately and works without a single problem. It also doesn't need any additional tweaks to the security context.

In short, deploying NPM as a custom app using the official image and quick start instructions, makes it start immediately and work without any issue. Installing the TrueNAS app takes forever to start, which results in constant liveness probe failures, and requires additional security context tweaking, which seems unneeded.

**Comment by krojew** [comment]
I think we need to look at it from an end user perspective. After installing the app with default settings, it doesn't seem to work unless one is ready to wait for an hour after deployment. When running the custom app, everything works. I'd argue, the default app installation should work no worse.

**Comment by stavros-k** [comment]
> I think we need to look at it from an end user perspective. After installing the app with default settings, it doesn't seem to work unless one is ready to wait for an hour after deployment. When running the custom app, everything works. I'd argue, the default app installation should work no worse.

The long installation time is usually when the apps pool is on HDDs.
Can you verify if that's also the case with your system?

But, I'm not sure we should lower security by default.

**Comment by krojew** [comment]
Yes, the apps use a 6 HDD pool.

**Comment by stavros-k** [comment]
Yea, the overlayfs does not seem to perform well on HDDs.
At least when there are that heavy I/Os as NPM does. 
It `chown`s a lot of directories, which at least one is probably having a **lot** of small files.

**Comment by krojew** [comment]
@stavros-k I tried the updated version and the same issue still exists when using 0 as the user/group.

**Comment by krojew** [comment]
Can confirm last fixes work.
